libraries/AP_Math/tests/test_math.cpp

```diff
@@ -277,6 +277,11 @@ TEST(MathTest, Square)
     AP_Float t_sqfloat;
     t_sqfloat = sq(2);
     EXPECT_EQ(4.f, t_sqfloat);
+
+    EXPECT_FLOAT_EQ(sq(2.3), 5.289999999999999);  // uses template sq
+    EXPECT_FLOAT_EQ(sq(2.3f), 5.29); // uses sq(float v)
```

Comment:
need a sq(very_large_number) check it is correct double number
maybe with pow(2,50) ?

Reply:
Is this holding up a merge?

Reply:
This was done; it's the last test in the list